src/demo/SdkMount.tsx

```diff
@@ -17,6 +17,10 @@ import * as Onfido from '../index'
 */
 
 const Onfido = window.Onfido
```

Comment:
For Auth you should be able to just do `const Onfido = window.OnfidoAuth` which works fine in the sample app but for some reason it doesn't in this demo app

Reply:
Interesting. I tried `import * as Onfido from '../index'` after talking to Carla about how to get it running locally & she suggested this, which worked. So just went with that.